pkg/storaged/utils.js

```diff
@@ -910,6 +911,15 @@ export function get_active_usage(client, path, top_action, child_action, is_temp
                     enter_unmount(children[c], c, false);
                 enter_unmount(block, mp, true);
             });
+        } else if (swap) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240118-122938-e6fc1c3f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)